crates/bevy_reflect/src/type_info.rs

```diff
@@ -72,7 +80,7 @@ use std::fmt::Debug;
 /// ```
 ///
 /// [utility]: crate::utility
-pub trait Typed: Reflect {
+pub trait Typed: PartialReflect {
```

Comment:
hmm. We should make a list of "should this be "PartialReflect" or "Reflect" somewhere so we can follow upp.

Reply:
actually i think this can be changed in this PR, since its been open for more than 6 months with very little design changes. in the meantime #6971 was merged which removed the `Typed` impls for all the `PartialReflect`-only types.